js/components/base-components/listItem.js

```diff
@@ -0,0 +1,51 @@
+var React = require('react');
+var Icon = require('../icon');
+var classnames = require('classnames');
+
+var listItem = React.createClass({
+  handleSelect: function () {
+    this.props.handleSelect(this.props.id);
+  },
+
+  render: function () {
+    var classes = {
+      listItem: 'oo-list-item',
+      icon: 'icon',
+      text: 'text'
+    };
+
+    if (this.props.selected) {
+      for (var key in classes){
+        classes[key] = classes[key] + ' select';
```

Comment:
We should be able to use the `classnames` utility for this purpose. It looks like it's required above, but not used.